lecture_1/lecture_1_passed/passed.go

```diff
@@ -1,50 +1,69 @@
 package main
 
 import (
+	//imports are now in a correct order
```

Comment:
These comments seem like they are responses to our comments on the PR - they shouldn't be in the code. Respond to PR comments on the github UI not in the code.

Reply:
Understood, fixed!